group-coordinator/src/main/resources/common/message/StreamsGroupCurrentMemberAssignmentValue.json

```diff
@@ -0,0 +1,46 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The streams rebalance protocol is in development. This schema is subject to non-backwards-compatible changes.
+{
+  "type": "data",
+  "name": "StreamsGroupCurrentMemberAssignmentValue",
+  "validVersions": "0",
+  "flexibleVersions": "0+",
+  "fields": [
+    { "name": "MemberEpoch", "versions": "0+", "type": "int32",
+      "about": "The current member epoch that is expected from the member in the heartbeat request." },
+    { "name": "PreviousMemberEpoch", "versions": "0+", "type": "int32",
+      "about": "If the last epoch bump is lost before reaching the member, the member will retry with the previous epoch." },
+    { "name": "State", "versions": "0+", "type": "int8",
+      "about": "The member state. See StreamsGroupMember.MemberState for the possible values." },
+    { "name": "ActiveTasks", "versions": "0+", "type": "[]TaskId",
+      "about": "Currently assigned active tasks for this streams client." },
+    { "name": "StandbyTasks", "versions": "0+", "type": "[]TaskId",
+      "about": "Currently assigned standby tasks for this streams client." },
+    { "name": "WarmupTasks", "versions": "0+", "type": "[]TaskId",
+      "about": "Currently assigned warming up tasks for this streams client." },
+    { "name": "ActiveTasksPendingRevocation", "versions": "0+", "type": "[]TaskId",
+      "about": "The active tasks that must be revoked by this member." }
+  ],
+  "commonStructs": [
+    { "name": "TaskId", "versions": "0+", "fields": [
+      { "name": "Subtopology", "type": "string", "versions": "0+",
+        "about": "The subtopology identifier." },
+      { "name": "Partitions", "type": "[]int32", "versions": "0+",
```

Comment:
Why is this an array?
I see that it is an array in `ConsumerGroupCurrentMemberAssignmentValue` but there it stores partitions per topic. Here we want to store a task ID and a task ID consists of a subtopology and one single partition number.